.eslintrc.cjs

```diff
@@ -1,7 +1,9 @@
 module.exports = {
+  parser: '@babel/eslint-parser',
```

Comment:
`@babel/eslint-parser` added for parsing top-level await syntax now allowed in ESM.